herpetologica.csl

```diff
@@ -0,0 +1,139 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-US">
+  <info>
+    <title>Herpetologica</title>
+    <id>http://www.zotero.org/styles/herpetologica</id>
+    <link href="http://www.zotero.org/styles/herpetologica" rel="self"/>
+    <link href="http://herpetologistsleague.org/author-guidelines/" rel="documentation"/>
+    <author>
+      <name>Pablo Melchor</name>
+      <uri>https://pablomelchor.com/hello/</uri>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="biology"/>
+    <issn>0018-0831</issn>
+    <summary>Citation style for Herpetologica</summary>
+    <updated>2016-11-14T12:00:00+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <macro name="editor">
+    <names variable="editor" delimiter=", " prefix="(" suffix=")">
+      <name and="text" initialize-with="." delimiter=", "/>
+      <label form="short" prefix=", "/>
+    </names>
+  </macro>
+  <macro name="author">
+    <names variable="author">
+      <name name-as-sort-order="first" and="text" sort-separator=", " initialize-with="." delimiter=", " delimiter-precedes-last="always"/>
+      <label form="short" prefix=" (" suffix=")"/>
+      <substitute>
+        <names variable="editor"/>
```

Comment:
for author-date styles, make the substitute elements for the author and author-short macros correspond (since you'll otherwise end up with non-matching citation-bibliography pairs). Either add translator here or remove it below in line 40.